tests/integration/rules/duo/duo_bypass_code_create_non_auto_generated.json

```diff
@@ -0,0 +1,35 @@
+{
+  "records": [
+    {
+      "data": {
+        "action": "bypass_create",
+        "description": "{\"bypass\": \"\", \"count\": 1, \"auto_generated\": false, \"valid_secs\": null, \"remaining_uses\": null, \"user_id\": \"...\"}",
+        "object": "...",
+        "timestamp": "1234567890",
+        "username": "..."
+      },
+      "description": "A DUO bypass code that was hand crafted should create an alert.",
+      "log": "duo:administrator",
+      "service": "stream_alert_app",
+      "source": "prefix_cluster_duo_auth_sm-app-name_app",
```

Comment:
should I create a new source? (need some guidance w/detail)

Reply:
@mime-frame yeah you can create a new source that would more accurately reflect this. Update sources.json to something like: 

```
...
"stream_alert_app": {
    "prefix_cluster_duo_auth_sm-app-name_app": {
      "logs": [
        "duo"
      ]
    },
    "prefix_cluster_duo_admin_sample-app-name_app": {
      "logs": [
        "duo"
      ]
    },
  }
...
```

Reply:
And then update your source here to reflect it